src/main/resources/db/migration/V202409231158__create_table_feedback.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE feedback (
+    id SERIAL PRIMARY KEY,
+    employee_id INT NOT NULL,
+    company_id INT NOT NULL,
+    date_of_feedback DATE DEFAULT current_date,
+    anonymity VARCHAR(255) NOT NULL,
+    reviewed BOOLEAN NOT NULL DEFAULT false,
+    feedback VARCHAR(255) NOT NULL,
+    response_id INT[]
+);
```

Comment:
There are no indexes in feedback or in response, think if you need any

Reply:
I'll add them. because there is no fk i'll have to join on different fields so it will be better to add it